resources/views/privacy.blade.php

```diff
@@ -13,29 +13,73 @@
+                <li>To request Georgia Tech's Institute Approved Absence forms on your behalf for qualifying activities</li>
+            </ul>
+
+            <h2>How does RoboJackets store your data?</h2>
+            <p>Data collected by RoboJackets is stored on a server secured within a Georgia Tech datacenter. Personally indentifiable information is encrypted at rest.</p>
+
+            <h2>How will RoboJackets share your data?</h2>
+            <p>RoboJackets shares data with certain external organizations in the course of conducting day to day business.</p>
+            <ul>
+                <li>Identidying information, such as GTID and your name will be shared as strictly necessary to Georgia Tech campus departments performing work on our behalf, including but not limited to, granting BuzzCard access to the shop, enrollment in SUMS, obtaining Institute Approved Absence letters, and reimbursement processing.</li>
+                <li>Resumes of all active members will be shared with our sponsors. You can opt out of this by not submitting a resume on this website.</li>
+                <li>Attendance data will be shared in aggregate or with identifying information to Georgia Tech campus departments.</li>
+                <li>Demographic information will be shared in aggregate to current and potential sponsors.</li>
+                <li>Information regarding an individual payment, including your email address, will be shared with Square, RoboJackets' designated payments processor, for the purposes of completing a credit/debit card transaction. You can opt-out of this sharing by using another payment method.
+                <li>All other information will be released only to RoboJackets officers in the course of conducting business.</li>
             </ul>
-            <p>RoboJackets will only release information about you as strictly necessary to Georgia Tech campus departments performing work on our behalf, including but not limited to granting BuzzCard access to the shop, enrollment in SUMS, and reimbursement processing. RoboJackets will release the resumes of all active members to our sponsors.</p>
-            <p>Demographic information may be reported in aggregate to current or potential sponsors.</p>
-            <p>All other information will be released only to RoboJackets officers in the course of conducting business.</p>
+            <p>RoboJackets does not sell any of the information collected to any organizations. However, sponsors who provide financial support to RoboJackets will receive aggregated data as outlined above as a benefit of their support.</p>
+
+            <h2>Cookies</h2>
+            <p>This application uses cookies to keep you logged in during and between sessions. If you elect to disable first-party cookies in your browser settings, this website may no longer function as intended.</p>
```

Comment:
Add commas (in, during, and between)

Reply:
It's not a list. We could hyphenate logged-in to improve clarity, perhaps.

Reply:
A hyphen would be good. I misread that twice.